src/Sulu/Bundle/DocumentManagerBundle/Tests/Functional/Bridge/DocumentManagerRegistryTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\DocumentManagerBundle\Tests\Functional\Bridge;
+
+use Sulu\Bundle\ContentBundle\Document\PageDocument;
+use Sulu\Bundle\TestBundle\Testing\SuluTestCase;
+
+class DocumentManagerRegistryTest extends SuluTestCase
+{
+    private $registry;
+
+    public function setUp()
+    {
+        $this->registry = $this->getContainer()->get('sulu_document_manager.registry');
+        $this->getContainer()->get('sulu_document_manager.initializer')->initialize(null, true);
+    }
+
+    /**
+     * It should be able to write to both the default and "live" workspace as configured
+     * in the TestBundle.
+     */
+    public function testWriteToDifferentSessions()
+    {
+        $dmDefault = $this->registry->getManager('default');
+        $dmLive = $this->registry->getManager('live');
```

Comment:
Would go with `$defaultDocumentManager` and  `$liveDocumentManager`, since we do not abbreviate variable names.